### PR TITLE
TINY-6633: Add HTML5 elements to default formats

### PR DESCRIPTION
**Pull request by ltrouton**
Related Ticket: TINY-6633

Description of Changes:
* Allow the `small` element to be removed when clearing text formatting
* Add `video` and `audio` to align formats. Worth noting that if the media plugin is enabled, the alignments won't work on them which needs to be looked into in a separate ticket.

Pre-checks:
* [X] Changelog entry added
* [X] Tests have been added (if applicable)
* ~~[ ] Branch prefixed with `feature/` for new features (if applicable)~~
* ~~[ ] License headers added on new files (if applicable)~~

Review:
* [X] Milestone set
* [x] Review comments resolved

GitHub issues (if applicable):

### CODE REVIEW

**Review by ashfordneil — APPROVED** [review]

**Review by JamesToohey — APPROVED** [review]

**Comment by JamesToohey** [comment]
Can we rename the PR given the new changes?

File: modules/tinymce/src/core/test/ts/browser/fmt/MediaAlignTest.ts [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+type Alignment = 'left' | 'center' | 'right' | 'justify';
+
+describe('browser.tinymce.core.delete.MediaAlignTest', () => {
+  const hook = TinyHooks.bddSetupLight<Editor>({
+    base_url: '/project/tinymce/js/tinymce'
+  }, [ Theme ], true);
+
+  const mediaApproxStructure = (tag: string, alignment: Alignment) => {
+    const alignStyles = (str: ApproxStructure.StringApi) => ({
```
Comment (ashfordneil) [inline modules/tinymce/src/core/test/ts/browser/fmt/MediaAlignTest.ts:17]: if `alignment` is already in scope here why not make this function just return the style assertion directly? Replace this delayed object indexing with a switch statement and some `return`s

**Review by ashfordneil — APPROVED** [review]

File: modules/tinymce/src/core/main/ts/fmt/DefaultFormats.ts [hunk 2]
```diff
@@ -189,7 +189,7 @@ const get = (dom: DOMUtils) => {
     ]
   };
 
-  Tools.each('p h1 h2 h3 h4 h5 h6 div address pre div dt dd samp'.split(/\s/), (name) => {
+  Tools.each('p h1 h2 h3 h4 h5 h6 div address pre dt dd samp'.split(/\s/), (name) => {
```
Comment (metricjs) [inline modules/tinymce/src/core/main/ts/fmt/DefaultFormats.ts:192]: I think we discussed this, but I've forgotten the reasoning sorry? Also, should this be mentioned in the changelog?

**Review by metricjs — APPROVED** [review]

File: modules/tinymce/src/core/main/ts/fmt/DefaultFormats.ts (hunk 2 above)
Comment (ltrouton) [reply modules/tinymce/src/core/main/ts/fmt/DefaultFormats.ts:192]: It's just removing the duplicate `div` from the list. There won't be any functionality change from removing the duplicate.

**Review by JamesToohey — APPROVED** [review]

File: modules/tinymce/src/core/test/ts/browser/fmt/MediaAlignTest.ts (hunk 1 above)
Comment (ltrouton) [reply modules/tinymce/src/core/test/ts/browser/fmt/MediaAlignTest.ts]: Done

**Pull request merged**